components/authz-service/engine/opa/opa.go

```diff
@@ -23,20 +23,16 @@ import (
 
 // State wraps the state of OPA we need to track
 type State struct {
-	log                    logger.Logger
-	store                  storage.Store
-	v2p1Store              storage.Store
-	queries                map[string]ast.Body
-	compiler               *ast.Compiler
-	modules                map[string]*ast.Module
-	partialAuth            rego.PartialResult
-	v2PreparedEvalProjects rego.PreparedEvalQuery
+	log                  logger.Logger
+	store                storage.Store
+	queries              map[string]ast.Body
+	compiler             *ast.Compiler
+	modules              map[string]*ast.Module
+	preparedEvalProjects rego.PreparedEvalQuery
 }
 
 // this needs to match the hardcoded OPA policy document we've put in place
 const (
-	authzQuery              = "data.authz.authorized"
-	filteredPairsQuery      = "data.authz.introspection.authorized_pair[_]"
 	authzProjectsV2Query    = "data.authz_v2.authorized_project[project]"
 	filteredPairsV2Query    = "data.authz_v2.introspection.authorized_pair[_]"
 	filteredProjectsV2Query = "data.authz_v2.introspection.authorized_project"
```

Comment:
More V2 removal at a minimum in the variable names (though the paths would be nice too):
```suggestion
	authzProjectsQuery    = "data.authz_v2.authorized_project[project]"
	filteredPairsQuery    = "data.authz_v2.introspection.authorized_pair[_]"
	filteredProjectsQuery = "data.authz_v2.introspection.authorized_project"
```